src/components/RoomView/OperativeBoard.tsx

```diff
@@ -55,8 +56,8 @@ const OperativeBoard = () => {
 
   return (
     <div style={style}>
-      {words.map((word) => {
-        return <Card key={word.id} word={word} />;
+      {words.map((word: CardObj) => {
+        return <Card key={word.id} word={word.word} id={word.id} isVisibleToAll={word.isVisibleToAll} wordString={word.wordString} wordId={word.wordId} boardId={word.boardId} teamId={word.teamId}  />;
```

Comment:
i guess typescript wanted ALL the types to be passes as props in this Card component, even though we aren't using all of them in Card technically